custom-words.txt

```diff
@@ -493,9 +494,8 @@ codegen
 codegeneration
 codemirror
 codepackage
-codespace
-codespaces
-coffeelake
+codesigning
+Coffeelake
```

Comment:
Why did you remove the codespace entries? Also please use lowercase.

Reply:
I must've messed up the rebase. Thanks for the catch, this is fixed.